### PR TITLE
UI/login controller

### PR DESCRIPTION
**Pull request by YasuyoshiWada**
CREATE LOGIN CONTROLER

### CODE REVIEW

**Comment by Ditzler01** [comment]
Please resolve the conflicts

File: app/Http/Controllers/Auth/LoginController.php
Comment (Ditzler01) [inline app/Http/Controllers/Auth/LoginController.php:1]: We don't need to modify the login controller.
Please communicate with Hikari-san on how did we fix the login controller.

File: database/migrations/2023_09_16_023955_add_columns_to_users_table.php [hunk 1]
```diff
@@ -14,10 +14,17 @@ class AddColumnsToUsersTable extends Migration
...
         Schema::table('users', function (Blueprint $table) {
-            $table->string('firstname');
-            $table->string('lastname');
-            $table->string('username')->unique();
+            $table->string('username')->after('name');
+            $table->enum('status', ['active', 'inactive'])->after('password')->default('active');
+            $table->foreignId('role_id')->constrained()->after('status')->nullable();
+            $table->string('address')->after('role_id')->nullable();
+            $table->string('phone_number')->after('address')->nullable();
+            $table->string('image')->after('phone_number')->nullable();
```
Comment (Ditzler01) [inline database/migrations/2023_09_16_023955_add_columns_to_users_table.php:22]: Replace the string with longText. It should look like this.

$table->longText('image')->after('phone_number')->nullable();

Comment (YasuyoshiWada) [reply database/migrations/2023_09_16_023955_add_columns_to_users_table.php]: I already finshed this.

**Pull request merged**